### DISCUSSION TITLE
deck.gl equivalent of maxbounds?

### DISCUSSION THREAD

**Discussion by arthurgailes** [visgl/deck.gl#6226] [Q&A] [answered]
Hello,

If possible, I'd like to apply (and re-apply) a maximum boundary to my map. I don't see this property in the docs; is there a way to establish it?

Thank you!

**Comment by kylebarron** [comment, marked as answer]
You mean to prevent a map from scrolling outside a given bounds? https://deck.gl/docs/developer-guide/interactivity#add-constraints-to-view-state

**Reply by arthurgailes** [reply]
Perfect, thanks very much!
